libs/datasets/timeseries.py

```diff
@@ -1,13 +1,13 @@
 import warnings
-from typing import List
+from typing import List, Optional
 import pandas as pd
 from libs import us_state_abbrev
 from libs.datasets import dataset_utils
 from libs.datasets import dataset_base
 from libs.datasets import custom_aggregations
 from libs.datasets.common_fields import CommonIndexFields
 from libs.datasets.common_fields import CommonFields
-from libs.datasets.dataset_utils import AggregationLevel
+from libs.datasets.dataset_utils import AggregationLevel, make_binary_array
```

Comment:
fwiw for functions especially, I'd rather import `dataset_utils` as a top level import.  makes it clear when reading the code where `make_binary_array` is coming from.

Reply:
Done! Ha, I've had reviewers push me the opposite direction in the past.